pkg/kubelet/volume_manager.go

```diff
@@ -19,21 +19,27 @@ package kubelet
 import (
 	"sync"
 
+	"k8s.io/kubernetes/pkg/api"
 	kubecontainer "k8s.io/kubernetes/pkg/kubelet/container"
 	"k8s.io/kubernetes/pkg/types"
 )
 
 // volumeManager manages the volumes for the pods running on the kubelet.
 // Currently it only does book keeping, but it can be expanded to
 // take care of the volumePlugins.
+// TODO(saad-ali): note that volumeManager will be completley refactored as part
+// of mount/unmount refactor.
 type volumeManager struct {
-	lock       sync.RWMutex
-	volumeMaps map[types.UID]kubecontainer.VolumeMap
+	lock         sync.RWMutex
+	volumeMaps   map[types.UID]kubecontainer.VolumeMap
+	volumesInUse []api.UniqueDeviceName
 }
 
 func newVolumeManager() *volumeManager {
-	vm := &volumeManager{}
-	vm.volumeMaps = make(map[types.UID]kubecontainer.VolumeMap)
+	vm := &volumeManager{
+		volumeMaps:   make(map[types.UID]kubecontainer.VolumeMap),
+		volumesInUse: []api.UniqueDeviceName{},
```

Comment:
Oh, neat idea. This code is also getting gutted in the mount/unmount redesign PR. But I'll make a note to consider it in that PR.